.travis.yml

```diff
@@ -1,10 +1,10 @@
 sudo: required
 os:
 - linux
-- osx
+# - osx
 language: node_js
 node_js:
-- node
+# - node
```

Comment:
I would include this (it uses latest version) or explicitly put 11.

Additionally, Node 6 EOL is at the end of the month, so we can go ahead and remove it.

Reply:
Done.